From c37921267f152c47e3ea357cede6d0ef76075f33 Mon Sep 17 00:00:00 2001
From: Han Xu <han.xu@nxp.com>
Date: Wed, 24 Jan 2018 15:58:12 -0600
Subject: [PATCH 3252/5242] MLK-17457: arm64: dts: enable usb otg for imx8mq
 ddr4 arm2

commit  b534362170dc7e0faae1d33bf65ce451d3713e2c from
https://source.codeaurora.org/external/imx/linux-imx.git

enable the usb otg for mfgtool download on imx8mq ddr4 arm2 board.

Signed-off-by: Han Xu <han.xu@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../boot/dts/freescale/fsl-imx8mq-ddr4-arm2.dts    |   13 +++++++++++++
 1 file changed, 13 insertions(+)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8mq-ddr4-arm2.dts b/arch/arm64/boot/dts/freescale/fsl-imx8mq-ddr4-arm2.dts
index bc299c7..b5248cf 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8mq-ddr4-arm2.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8mq-ddr4-arm2.dts
@@ -342,6 +342,19 @@
 	status = "okay";
 };
 
+&usb3_phy0 {
+	status = "okay";
+};
+
+&usb3_0 {
+	status = "okay";
+};
+
+&usb_dwc3_0 {
+	status = "okay";
+	dr_mode = "peripheral";
+};
+
 &wdog1 {
 	pinctrl-names = "default";
 	pinctrl-0 = <&pinctrl_wdog>;
-- 
1.7.9.5

